### PR TITLE
docs(sh-manage): add logging to production setup

### PR DESCRIPTION
**Pull request by mffap**
dependency of #5747

### CODE REVIEW

File: docs/docs/self-hosting/manage/production.md [hunk 1]
```diff
@@ -45,6 +45,24 @@ Tracing:
...
+ZITADEL follows the principles that guide cloud-native and twelve factor applications.
+Logs are a stream of time-ordered events collected from all running processes.
+
+ZITADEL processes write the following events to the standard output:
+
+- ZITADEL Runtime
+- Access (http calls)
+- Actions Runtime
+- Standard Output from Actions
+- Standard Output from Events
```
Comment (fforootd) [inline docs/docs/self-hosting/manage/production.md:59]:
```suggestion
- Runtime Logs (startup events, runtime errors, database connection changes)
- Access Logs (API request)
- Actions Execution Logs (success and error messages)
- Actions Logger (Actions can log customer chosen logs)
- Events (Operations who change resources, i.e create a user)
```

Comment (eliobischof) [reply docs/docs/self-hosting/manage/production.md:59]: Events are not logged to stdout.
I'd also give a hint where to turn them on or off depending on the defaults. I'll suggest sth.

Comment (mffap) [reply docs/docs/self-hosting/manage/production.md:59]: I can wrap Events into Actions Logs since they can be logged as part of an Action, right?

Comment (mffap) [reply docs/docs/self-hosting/manage/production.md:59]: > Action execution and actions logger feel a bit redundant. I would remove actions logger out.

Would be more concise. Personally I actually like the separation, so we can make it more explicit that also custom output can be sent to logs. This is very important in cases where you have a downstream system capturing events, for example (this is why I made this PR :) )

Comment (eliobischof) [reply docs/docs/self-hosting/manage/production.md]: We can't log ANY eventstore events to stdout, yet.
This is coming for ALL events with #5101

Comment (eliobischof) [reply docs/docs/self-hosting/manage/production.md]:
```suggestion
- Runtime Logs: Define the log level and record format [in the Log configuration section](https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml#L1-L4)
- Access Logs: Enable logging all HTTP and gRPC responses from the ZITADEL binary [in the LogStore section](https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml#L366) 
- Actions Exectution Logs: Actions can emit custom logs at different levels. For example, a log record can be emitted each time a user is created or authenticated. If you don't want to have these logs in STDOUT, you can disable this [in the LogStore section](https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml#L387) .
```

File: docs/docs/self-hosting/manage/production.md [hunk 2]
```diff
@@ -45,6 +45,23 @@ Tracing:
   MetricPrefix: zitadel
 ```
 
+## Logging
```
[the remark this answers is not in the document]
Comment (eliobischof) [reply docs/docs/self-hosting/manage/production.md:48]: added a suggestion above

**Pull request merged**